extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/JsonObject.java

```diff
@@ -0,0 +1,170 @@
+/*
+ Copyright (c) 2002 JSON.org
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ The Software shall be used for Good, not Evil.
```

Comment:
Unfortunately, this makes it non-free software. We can't include it.

Reply:
Better use Jackson here

Reply:
> Better use Jackson here

Ah wouldn't that meaning pulling Jackson to the core?

Reply:
Ah I missed @cescoffier comments in https://github.com/quarkusio/quarkus/pull/7430#discussion_r384390220, Jackson is not an option here